src/renderer/scss/component/_shapeshift.scss

```diff
@@ -0,0 +1,63 @@
+.shapeshift__wrapper.shapeshift__initial-wrapper {
+  min-height: 311px;
+}
+
+.shapeshift__content {
+  .spinner {
+    margin-top: 75px;
+  }
+}
+
+.shapeshift__success {
+  color: var(--color-primary);
+}
+
+.shapeshift__error {
+  color: var(--color-error);
```

Comment:
Is a `shapeshift__error` sufficiently different from other errors that it needs it's own styling? Same for success.

Reply:
Don't need this class. The success class is just to add the green color on the text. There isn't any styles that only do that, maybe it should be a more generic className?